.changeset/twelve-windows-give.md

```diff
@@ -0,0 +1,16 @@
+---
+'@graphql-codegen/typed-document-node': minor
```

Comment:
I marked these as `minor` since I viewed this as a feature, but feel free to correct this if this seems off!

Reply:
Thanks for confirming! I figured the list was too much so thanks for the assist. Updated in [5790112](https://github.com/dotansimha/graphql-code-generator/pull/10163/commits/57901125dac87c8e77adb6dcc0e309eef60f2994)